odroid_node:
  Motor: 0
  MotorWarmup: 0
controller:
  mode: 1
  g: 9.81
  m: 1.574 #1.7 with sensors
  J: [9.773E-3, 0.0, 0.0,
    0.0, 9.773E-3, 0.0,
    0.0,  0.0,  1.749E-2]
  del_t: 0.01
  R_conv: [1.0, 0.0, 0.0,
        0.0, -1.0, 0.0,
        0.0, 0.0, -1.0] # Convention conversion (NEU->NED)
  l: 0.305 # arm length
  c_tf: 0.1 # moment constant
  gain:
    att:
      kp: 0.5
      kd: 0.2
      ki: 0.01
      c: 0.10
    pos:
      kp: 6.5
      kd: 4.0
      ki: 0.01
      c: 0.10
  saturation:
    x: 0.1
    R: 0.01
